apps/mongodb/package.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
@yisding i saw that there are examples in `apps` and `examples` - where is the better place for this example?